frontend/src/components/BookingCard.tsx

```diff
@@ -0,0 +1,168 @@
+  margin-bottom: 0em;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 14px;
+  overflow: hidden;
+  color: #ff705c;
+`
+
+const TimeLabel = styled.div`
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 14px;
+`
+const BookingCCAName = styled.div`
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 14px;
+`
+
+const BookingRightDisplay = styled.div`
+  position: absolute;
+  right: 15px;
+  top: 50%;
+  transform: translate(-17%, -50%);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+const Icon = styled.img`
+  padding: 20px;
+`
+export default function BookingCard() {
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const params = useParams<{ facilityID: string }>()
+  const {
+    ViewStartDate,
+    ViewEndDate,
+    createSuccess,
+    createFailure,
+    isLoading,
+    facilityBookings,
+    selectedFacilityName,
+    selectedFacilityId,
+  } = useSelector((state: RootState) => state.facilityBooking)
+  const getHumanReadableTime = (eventStartTime: number) => {
+    const date = new Date(eventStartTime * 1000)
+    let hour = date.getHours().toString()
+    if (hour.length == 1) {
+      hour = '0' + hour
+    }
+    let minutes = date.getMinutes().toString()
+    if (minutes.length == 1) {
+      minutes = '0' + minutes
+    }
+
+    return hour + minutes
+  }
+
+  const getHumanReadableDate = (eventTime: number) => {
+    const date = new Date((eventTime + 28800) * 1000)
+    const numday = date.getUTCDate()
+    const monthInt = date.getUTCMonth() + 1
+    const day = date.getUTCDay()
+
+    return numday + '/' + monthInt
+  }
+
+  const fetchTelegram = async (booking) => {
+    try {
+      fetch(DOMAIN_URL.FACILITY + ENDPOINTS.TELEGRAM_HANDLE + '/' + booking.userID, {
+        method: 'GET',
+        mode: 'cors',
+      })
+        .then((resp) => resp.json())
+        .then((data) => {
+          if (data.telegramHandle === '' || data.telegramHandle === undefined) {
+            console.log(data.err)
+          } else {
+            openTelegram(data.telegramHandle)
+          }
+        })
+    } catch (err) {
+      console.log(err)
+    }
+  }
+  const openTelegram = (userID) => {
+    const site = 'https://telegram.me/' + userID
+    window.open(site)
+  }
+
+  return (
+    <BookingGroup>
+      {facilityBookings?.map((event) => (
+        <BookingContainer
+          key={event.bookingID}
+          onClick={() => {
+            history.push(PATHS.VIEW_FACILITY_BOOKING_ID + event.bookingID)
+          }}
+        >
+          {/* <EventAvatar src={dummyAvatar} /> */}
+          <BookingLeftDisplay>
+            <DateLabel>
+              <b>
+                {days[ViewStartDate.getDay()]}{' '}
+                {ViewStartDate.getDate() + ' ' + months[ViewStartDate.getMonth()] + ' ' + ViewStartDate.getFullYear()}
+              </b>
+            </DateLabel>
+            <TimeLabel>
+              <b>
```

Comment:
this too, consider making it bold under TimeLabel instead of wrapping with b

Reply:
Resolved in commit 6c4688c5a5c1853aa8ff09a0d49fdb749f87bdbb